pkg/ratelimit/config.go

```diff
@@ -71,9 +71,8 @@ func RateLimiterFor(ctx context.Context, c *Config) (limiter.Store, error) {
 		}
 
 		return redisstore.NewWithPool(config, &redis.Pool{
-			DialContext: func(ctx context.Context) (redis.Conn, error) {
-				return redis.DialContext(ctx, "tcp", addr,
-					redis.DialUsername(c.RedisUsername),
+			Dial: func() (redis.Conn, error) {
+				return redis.Dial("tcp", addr,
```

Comment:
Please continue to use DialWithContext https://godoc.org/github.com/opencensus-integrations/redigo/redis#DialWithContext